### PR TITLE
Line and bar chart

### PR DESCRIPTION
**Pull request by Basiol97**
Integrated the Chart-js to display the fetched data in the form of line and bar charts.

### CODE REVIEW

File: src/components/Pages/LoadingPage/components/LandingPage.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            return Math.abs(Number(price));
+        }
+    }
+
+    getCurrentDate = () => {
+        const today = new Date(this.state.data.prices?.[this.state.data.prices?.length - 1][0]);
+        const dd = String(today.getDate()).padStart(2, '0');
+        const mm = (today.toLocaleString('default', { month: 'long' })).slice(0, 3);
+        const yyyy = today.getFullYear();
+        const currentDate = mm + " " + dd + ', ' + yyyy;
```
Comment (yoloOnTheBattlefield) [inline src/components/Pages/LoadingPage/components/LandingPage.js:54]: const currentDate = `${mm } " " + dd + ', ' + yyyy;

File: src/components/Pages/LoadingPage/components/LandingPage.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            const currencySymbols = "mln";
+            return formatedPrice + currencySymbols;
+        } else if (Math.abs(Number(price)) >= 1.0e+3) {
+            const formatedPrice = (Math.abs(Number(price)) / 1.0e+3).toFixed(3);
+            const currencySymbols = "k";
+            return formatedPrice + currencySymbols;
+        } else {
+            return Math.abs(Number(price));
+        }
+    }
```
Comment (yoloOnTheBattlefield) [inline src/components/Pages/LoadingPage/components/LandingPage.js:47]: add in a separate file

File: src/components/Pages/LoadingPage/components/Line.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        {this.props.isLoading && <div></div>}
+        {hasUSer && (
+          <div >
+            <LineChart formatPrice={this.props.formatPrice} getCurrentDate={this.props.getCurrentDate} currentPriceValue={currentPriceValue} chartData={chartData} options={options} />
+          </div>
+        )}
+      </div>
+    );
+  }
+}
```
Comment (yoloOnTheBattlefield) [inline src/components/Pages/LoadingPage/components/Line.js:85]: make this a functional component

File: src/components/Pages/LoadingPage/styled/LandingPage.styled.js [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+export const Button = styled.button`
+flex: 1;
+border-radius: 15px;
+padding: 5px 10px;
+color: white;
+border: none;
+font-family: 'Poppins', sans-serif;
+cursor: pointer;
+background-color: ${props => props.primary ? "#2172E5" : " #1F2128"};;
+`;
```
Comment (yoloOnTheBattlefield) [inline src/components/Pages/LoadingPage/styled/LandingPage.styled.js:158]: indent

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/Pages/LoadingPage/components/LandingPage.js (hunk 1 above)
Comment (Basiol97) [reply src/components/Pages/LoadingPage/components/LandingPage.js:54]: Resolved by implementing moment js

File: src/components/Pages/LoadingPage/components/Line.js (hunk 3 above)
Comment (Basiol97) [reply src/components/Pages/LoadingPage/components/Line.js:85]: Resolved

**Pull request closed without merging**